tests/Doctrine/Tests/Models/PriceRegions/PriceRegion.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Doctrine\Tests\Models\PriceRegions;
+
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Table;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\ORM\Mapping\ManyToOne;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\JoinColumns;
+use Doctrine\ORM\Mapping\JoinColumn;
+use Doctrine\ORM\Mapping\Cache;
+
+/**
+ * @Entity
+ * @Table(name="price_regions")
+ * @Cache
+ */
+class PriceRegion
+{
+    /**
+     * @Id
+     * @Column(type="string", length=25)
+     */
+
+    public $id;
+
+    /**
+     * @Id
+     * @Column(type="string", length=25)
+     */
+
+    public $master_user_id;
```

Comment:
This mapping is invalid: you are re-using the `master_user_id` column in both a field an an association